### PR TITLE
{bio}[foss/2020a,GCC/8.3.0] MaSuRCA v4.01, Flye v2.8.2, bpp v4.3.8

### PR DESCRIPTION
**Pull request by stevecdavis**
Both software updated to current versions
Flye added as dependency to MaSuRCA

### CODE REVIEW

File: easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.4.2-foss-2020a-Perl-5.30.2.eb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+buildopts = "install-special"
+start_dir = "global-1"
+
+postinstallcmds = [
+    # fix location of 'bin' in install prefix in runCA and runCA-dedupe scripts
+    # escaping single quotes within single quotes is tricky, so we use $'...' to use ANSI C-like escaping
+    "sed -i $'s|^$bin =.*|$bin = \"$ENV{\'EBROOTMASURCA\'}/bin\";|g' %(installdir)s/bin/runCA",
+    "sed -i $'s|^$bin =.*|$bin = \"$ENV{\'EBROOTMASURCA\'}/bin\";|g' %(installdir)s/bin/runCA-dedupe",
+    # fix hardcoded path in masurca script, just point back to 'bin' subdirectory instead
+    "sed -i 's@../CA8/Linux-amd64/bin@../bin@g' %(installdir)s/bin/masurca",
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.4.2-foss-2020a-Perl-5.30.2.eb:53]: `W299 trailing whitespace`

**Review by migueldiascosta — DISMISSED** [review]
@stevecdavis thank you for your contribution! you may want to have a look at `eb --check-contrib` to check style issues before submitting/committing

**Comment by migueldiascosta** [comment]
and for subsequent PRs, `--new-pr` et al (https://docs.easybuild.io/en/develop/Integration_with_GitHub.html)

**Comment by akesandgren** [comment]
Flye should be moved down to GCC level, see https://github.com/stevecdavis/easybuild-easyconfigs/pull/1

**Comment by boegel** [comment]
@stevecdavis You've now pushed a `bpp` easyconfig in this PR too, which is unintended I think?
Any changes you push into your `develop` branch will get added to this PR...

We can looking into getting this merged as-is (with `bpp` included) before the situation "escalates", but going forward I strongly recommend using a separate branch per PR, and looking into the GitHub integration we have in EasyBuild (see https://docs.easybuild.io/en/latest/Integration_with_GitHub.html).

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @stevecdavis!

**Pull request merged**